src/containers/Home/total/total.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from "@testing-library/react";
+import Total from "./index";
+import configureStore from "../../Redux/Store";
+import { Provider } from "react-redux";
+
+const store = configureStore();
+import React from "react";
+
+test("renders learn react link", () => {
+  // substring match, ignore case
+
+  render(
+    <Provider store={store}>
+      <Total totalJobs={100} />
+    </Provider>
+  );
+  const totalElement = screen.getByText(/100 IT Job For Chất Developer/i);
+  expect(totalElement).toHaveTextContent(/100 IT Job For Chất Developer/i);
```

Comment:
We are testing "100 IT Job For Chất Developer" is visible, we cannot get it and test if it has the text.
Just test the text is visible
expect(screen.getByText(/100 IT Job For Chất Developer/i)).toBeVisible();

Reply:
thanks, i changed code.